bisqapps/shared/presentation/build.gradle.kts

```diff
@@ -87,6 +87,10 @@ kotlin {
             implementation(libs.androidx.lifecycle.runtime.compose)
             implementation(libs.logging.kermit)
             implementation(libs.kotlinx.coroutines)
+
+            implementation(libs.koin.core)
+            implementation(libs.koin.compose)
+            implementation(libs.koin.compose.viewModel)
```

Comment:
I don't think we need the view model dependency. Please see https://github.com/bisq-network/bisq-mobile/blob/main/bisqapps/README.md#app-architecture-design-choice .
The other 2 are great and will be greatly used for injecting the presenters

Reply:
Added viewmodel because of compose, no worries would demo code